charts/victoria-metrics-auth/README.md

```diff
@@ -15,73 +14,90 @@ Victoria Metrics Auth - is a simple auth proxy and router for VictoriaMetrics.
 
 Access a Kubernetes cluster.
 
-Add a chart helm repository with follow commands:
-
- - From HTTPS repository
+### Setup chart repository (can be omitted for OCI repositories)
 
-   ```console
-   helm repo add vm https://victoriametrics.github.io/helm-charts/
-
-   helm repo update
-   ```
- - From OCI repository
-  
-   ```console
-   helm repo add vm oci://ghcr.io/victoriametrics/helm-charts/
+Add a chart helm repository with follow commands:
 
-   helm repo update
-   ```
+```console
+helm repo add vm https://victoriametrics.github.io/helm-charts/
 
-List versions of ``vm/victoria-metrics-auth`` chart available to installation:
+helm repo update
+```
+List versions of `vm/victoria-metrics-auth` chart available to installation:
 
 ```console
 helm search repo vm/victoria-metrics-auth -l
 ```
 
-Export default values of ``victoria-metrics-auth`` chart to file ``values.yaml``:
+### Install `victoria-metrics-auth` chart
 
-```console
-helm show values vm/victoria-metrics-auth > values.yaml
-```
+Export default values of `victoria-metrics-auth` chart to file `values.yaml`:
+
+  - For HTTPS repository
+
+    ```console
+    helm show values vm/victoria-metrics-auth > values.yaml
+    ```
+  - For OCI repository
+
+    ```console
+    helm show values oci://ghcr.io/victoriametrics/helm-charts/victoria-metrics-auth > values.yaml
+    ```
 
 Change the values according to the need of the environment in ``values.yaml`` file.
 
 Test the installation with command:
 
-```console
-helm install vmauth vm/victoria-metrics-auth -f values.yaml -n NAMESPACE --debug --dry-run
-```
+  - For HTTPS repository
+
+    ```console
+    helm install vma vm/victoria-metrics-auth -f values.yaml -n NAMESPACE --debug --dry-run
+    ```
+
+  - For OCI repository
+
+    ```console
+    helm install vma oci://ghcr.io/victoriametrics/helm-charts/victoria-metrics-auth -f values.yaml -n NAMESPACE --debug --dry-run
+    ```
 
 Install chart with command:
 
-```console
-helm install vmauth vm/victoria-metrics-auth -f values.yaml -n NAMESPACE
-```
+  - For HTTPS repository
+
+    ```console
+    helm install vma vm/victoria-metrics-auth -f values.yaml -n NAMESPACE
+    ```
+
+  - For OCI repository
+
+    ```console
+    helm install vma oci://ghcr.io/victoriametrics/helm-charts/victoria-metrics-auth -f values.yaml -n NAMESPACE
+    ```
 
 Get the pods lists by running this commands:
 
 ```console
-kubectl get pods -A | grep 'vmauth'
+kubectl get pods -A | grep 'agent'
```

Comment:
Looks like a copy-paste typo for all grep 'agent' occurrences.

Reply:
thanks, fixed